apps/questapi/moon.yml

```diff
@@ -0,0 +1,19 @@
+tasks:
+  install:
+    command:
+      npm install --save-dev
+  dev:
+    command: 
+      npm run dev
+  build:
+    command:
+      npm run build
+  start:
+    command:
+      npm start
+  generateprismaclient:
+    command:
+      npx prisma generate
+  migratedb:
```

Comment:
as the review comment earlier, I suggest we use `db:migrate` instead of `migratedb`